pom.xml

```diff
@@ -18,15 +18,14 @@
 		<jdk.version>17</jdk.version>
 		<cds.services.version>2.1.0</cds.services.version>
 		<spring.boot.version>3.1.2</spring.boot.version>
-		<cloud.sdk.version>4.18.0</cloud.sdk.version>
-		<xsuaa.version>3.1.0</xsuaa.version>
+		<xsuaa.version>3.0.1</xsuaa.version>
 		<cf-java-logging-support.version>3.7.0</cf-java-logging-support.version>
-		<cds.install-cdsdk.version>7.0.3</cds.install-cdsdk.version>
+		<liquibase.version>4.20.0</liquibase.version>
+		<testcontainers.version>1.18.3</testcontainers.version>
```

Comment:
Liquibase and Testcontainers don't need to be managed, as they come with the Spring Boot dependencies BOM. We should use versions from there.